changelog/v0.24.3/helm-scanning.yaml

```diff
@@ -0,0 +1,6 @@
+changelog:
+- type: NON_USER_FACING
```

Comment:
nit: Since this is an internal library, the "users" are more our devs teams. In that light, I almost view this as a new feature, one which didn't exist before and devlopers upgrading to this library version can use